vignettes/quickstart.Rmd

```diff
@@ -26,33 +26,6 @@ then
 conn <- data.world()
 ```
 
-## Querying
```

Comment:
If possible, I'd like to prioritize Query over CRUD. In other words, I'd talk about query in the main page and move CRUD to a secondary page, if that makes sense. Also, we should include Windows instructions, if possible.